floe/restapi.py

```diff
@@ -4,8 +4,10 @@
     FloeDeleteException, FloeConfigurationException, FloeInvalidKeyException, \
     FloeOperationalException, FloeReadException
 from .helpers import sanitize_key
-from multiprocessing.pool import ThreadPool
+from concurrent.futures import ThreadPoolExecutor
 
+FLOE_REST_TIMEOUT = 30
+FLOE_TASK_TIMEOUT = 60
```

Comment:
I'm curious why these specific values?

Reply:
There was a bit of guess work, but my reasoning is to pick a long enough timeout that if we haven't received a response we can assume one isn't coming.  The API typically responds in under 50ms with maximums around 1s so 30s would be extremely out of character

The task timeout is twice as long because requests uses the timeout value as a connect and idle timeout, so worst case 30seconds to connect and 30s with no response is 60s

It could also be worthwhile to make these values tunable somehow so we can adjust them on the fly